### PR TITLE
Add k8s manifests and dev tooling

### PR DESCRIPTION
**Pull request by khos2ow**
#### Summary

Adding Kubernetes manifests and all the required developer toolings to be used on workstations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,17 +10,36 @@ Matterbuild is an internal Mattermost chatops tool for generating Mattermost rel
...
+Essentials:
+
 1. Install [Go](https://golang.org/doc/install)
 2. `brew install gnu-tar` for macOS
 
+Optionals:
+
+1. [Tilt](https://tilt.dev/) (to deploy on a local dev K8s cluster)
+2. [kind](https://kind.sigs.k8s.io/) (to spin up a local dev K8s cluster)
+
```
Comment (metanerd) [inline README.md:22]: I guess it is missing kustomize?

Comment (metanerd) [reply README.md:22]: Please add `config.json` to the readme.

**Review by metanerd — CHANGES_REQUESTED** [review]
It is beautiful! So convenient, I really like it!

File: README.md (hunk 1 above)
Comment (khos2ow) [reply README.md:22]: done

**Review by metanerd — APPROVED** [review]
Nice!

**Pull request merged**